### PR TITLE
add rapidfuzz

### PR DESCRIPTION
**Pull request by maxbachmann**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/rapidfuzz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: db0cd3b4cc024075362be92e247c0c2c7998bdbf9f775de4601717ba70b1bed9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```
Comment (synapticarbors) [inline recipes/rapidfuzz/meta.yaml]: it looks like this package is py3 only. If so, add in the `build` section under `script`:

```
skip: True  # [py2k]
```

File: recipes/rapidfuzz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - fuzzywuzzy.process
+
+about:
+  home: "https://github.com/rhasspy/rapidfuzz"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "rapid fuzzy string matching"
+  doc_url:
+  dev_url:
```
Comment (synapticarbors) [inline recipes/rapidfuzz/meta.yaml]: make `dev_url` the same as `home` and remove `doc_url`.

File: recipes/rapidfuzz/meta.yaml (hunk 1 above)
Comment (maxbachmann) [reply recipes/rapidfuzz/meta.yaml]: thanks this is my first package and I am still a bit clueless :)

**Comment by maxbachmann** [comment]
@synapticarbors how can I specify that a C++ compiler with C++17 support is required?

**Comment by synapticarbors** [comment]
pinging @conda-forge/help-python-c for assistance

**Comment by mariusvniekerk** [comment]
Depending on how the setup.py works you might want to try a build_ext before the pip just so that you can get some useful logs?

**Comment by maxbachmann** [comment]
https://github.com/conda-forge/staged-recipes/pull/11165 is a new PR that adds the same package

**Pull request closed without merging**